frontend/src/components/DrawerDetails/ResourceDetailsDrawer.tsx

```diff
@@ -0,0 +1,152 @@
+
+                setResourceDetails(combinedDetails);
+                console.log(combinedDetails);
+
+            }
+        } catch (error: unknown) {
+            if (error instanceof Error) {
+                console.error("Error fetching resource details:", error.message);
+            } else {
+                console.error("Unknown error:", error);
+            }
+            setResourceDetails(null);
+        }
+        setFetching(false);
+    };
+
+    useEffect(() => {
+        if (visible && record) {
+            if ("namespace" in record && "name" in record) {
+                const namespace = record.namespace as string;
+                const resourceName = record.name as string;
+                fetchResourceData(resourceType, resourceName, namespace);
+            }
+        }
+    }, [visible, record]);
+
+    const renderObject = (obj: any, parentKey = ""): React.ReactNode => {
+        if (!obj || typeof obj !== "object") return null;
+
+        return Object.entries(obj).map(([key, value]) => {
+            const panelKey = parentKey ? `${parentKey}.${key}` : key;
+
+            if (value && typeof value === "object") {
+                return (
+                    <Collapse key={panelKey} className={styles.collapse}>
+                        <Panel header={key} key={panelKey}>
+                            {renderObject(value, panelKey)}
+                        </Panel>
+                    </Collapse>
+                );
+            } else {
+                return (
+                    <div key={panelKey} className={styles.singleDetail}>
+                        <div className={styles.key}>{key}:</div>
+                        <Tooltip title={value?.toString() || "—"}>
+                            <div className={styles.value}>{value?.toString() || "—"}</div>
+                        </Tooltip>
+                    </div>
+                );
+            }
+        });
+    };
+
+    const renderTitle = () => {
+        if (record && "name" in record) {
+            return `${resourceType}: ${record.name}`;
+        }
+        return `${resourceType}: Unknown`;
+    };
+
+    return (
+        <Drawer
+            title={renderTitle()}
+            placement="right"
+            width={600}
+            onClose={onClose}
+            open={visible}
+            mask={false}
+        >
+            {fetching ? (
+                <Spin
+                    size="large"
+                    style={{ display: "block", textAlign: "center", marginTop: "100px" }}
+                />
+            ) : resourceDetails ? (
+                <div className={styles.detailsContainer}>
+                    {resourceType === "Helm" ? (
+                        <Collapse>
+                            {/* Секция с данными релиза */}
+                            <Panel header="Release" key={""}>
+                                {renderObject(resourceDetails[0])}
+                            </Panel>
```

Comment:
It would be helpful to automatically open details. Like this.

![image](https://github.com/user-attachments/assets/ac651569-833c-4317-bb3a-e42023653f22)

Reply:
Why?